api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -498,13 +498,13 @@ public List<LeonardoListPersistentDiskResponse> listPersistentDiskByProjectCreat
   public List<LeonardoListPersistentDiskResponse> listDisksByProjectAsService(
       String googleProject) {
 
-    DisksApi disksApi = disksApiProvider.get();
+    DisksApi disksApi = serviceDisksApiProvider.get();
     return leonardoRetryHandler.run(
         (context) ->
             disksApi.listDisksByProject(
                 googleProject,
                 null,
-                /*includeDeleted*/ true,
+                /*includeDeleted*/ false,
```

Comment:
I wasn't sure if this was helpful or not. I suppose that we could always parameterize this in the future, if people want to see deleted disks.